src/net.cpp

```diff
@@ -1215,14 +1217,8 @@ void CConnman::DisconnectNodes() {
             }
         }
     }
-    {
-        LOCK(cs_vNodes);
-        vNodesSize = vNodes.size();
-    }
-    if(vNodesSize != nPrevNodeCount) {
-        nPrevNodeCount = vNodesSize;
-        if(clientInterface)
-            clientInterface->NotifyNumConnectionsChanged(vNodesSize);
+    if(clientInterface && vNodesSize != nPrevNodeCount) {
```

Comment:
Commit "Improve lock handling in DisconnectNodes."

nit, space after `if`.

Reply:
fixed